ui/apps/platform/eslint-plugins/pluginPatternFly.js

```diff
@@ -0,0 +1,71 @@
+/* globals module */
+
+const rules = {
+    // ESLint naming convention for positive rules:
+    // If your rule is enforcing the inclusion of something, use a short name without a special prefix.
+
+    // ESLint naming convention for negative rules.
+    // If your rule only disallows something, prefix it with no.
+    // However, we can write forbid instead of disallow as the verb in description and message.
+
+    'no-Variant': {
+        // Replace Variant enum member with corresponding string literal.
+        // Because TypeScript string enumeration is source of truth for prop.
+        // For example, replace variant={AlertVariant.info} with variant="info"
+        meta: {
+            type: 'problem',
+            docs: {
+                description: 'Replace Variant enum member with corresponding string literal',
+            },
+            schema: [],
+        },
+        create(context) {
+            // Temporary forbid list limits changed files per contribution.
```

Comment:
The explicit forbid list seems like a balance between:
- Requiring the team to add new PatternFly string enum variants as they are encountered or added to PF6
- The risk of PF6 adding a `/\w+Variant/` import that _isn't_ a string enum used as a prop that would require a change here

Reply:
Deleted forbid list after rebase on separated contributions for:
* `AlertVariant`
* `ButtonVariant`
* `ModalVariant` although missed one